pkg/storage/consistency_queue.go

```diff
@@ -102,19 +102,30 @@ func (q *consistencyQueue) process(
 	if q.interval() <= 0 {
 		return nil
 	}
+
+	// Update the last processed time for this queue. We do this before actually
+	// processing because this queue will do it even in the case of an error
+	// (since a failed processing run could be caused by down followers).
```

Comment:
It's not clear from the comment why this is worth doing specifically for the consistency queue but not for any other queues.

Reply:
I'm really just backfilling rationale here, but this is the previous behavior. I'm going to add a bit of doubt to my comment to not make it seem like this was a decision prompted by an observable problem.